src/web_mercator.jl

```diff
@@ -0,0 +1,78 @@
+"""
+    WebMercatorfromLLA(datum=wgs84)
+
+Convert from LLA to Web Mercator / Pseudo Mercator, following the convention of
+proj, which uses a scaling factor of the semi-major axis of the ellipsoid
+https://proj.org/operations/projections/webmerc.html.
+
+!!! warning
+    For web mapping applications this projection is ubiquitous due to its
+    simplicity of implementation, but this simplicity gives rise to poor
+    mathematical properties: it doesn't preserve angles away from the equator.
+    Other projections should be preferred if possible, and especially when
+    measurement is important. See
+    https://earth-info.nga.mil/GandG/wgs84/web_mercator/(U)%20NGA_SIG_0011_1.0.0_WEBMERC.pdf
+    for an extended discussion.
+"""
+struct WebMercatorfromLLA <: Transformation
+    el::Ellipsoid
+end
+
+WebMercatorfromLLA(d::Datum=wgs84) = WebMercatorfromLLA(ellipsoid(d))
+
+"""
+    LLAfromWebMercator
+
+Inverse of WebMercatorfromLLA — see the docs for that transformation.
+"""
+struct LLAfromWebMercator <: Transformation
+    el::Ellipsoid
+end
+
+LLAfromWebMercator(d::Datum=wgs84) = LLAfromWebMercator(ellipsoid(d))
+
+
+function (trans::WebMercatorfromLLA)(lla::LLA)
+    xy = trans(LatLon(lla.lat, lla.lon))
+    SA[xy[1], xy[2], lla.alt]
+end
+
+function (trans::WebMercatorfromLLA)(ll::LatLon)
+    lat_lim = 85.6 # according to https://epsg.io/3857
```

Comment:
I think it's 85.06 or more exactly 

    const MAX_LAT = rad2deg(2*atan(ℯ^pi)-pi/2)

Reply:
Yes this was a typo - thanks!

We could use `rad2deg(2*atan(ℯ^pi)-pi/2)`, that's easy enough. (This is the value which makes the bounding box in northing and easting square, right?)

However I wondered whether the more permissive 85.06 was an intentional choice chosen by the standards body, in order to
* Use a rational rather than irrational number
* Allow some leeway for computing with geometries which have been clipped to the extent of the projection, but are slightly over after numerical error.